src/dvclive/studio.py

```diff
@@ -66,3 +68,11 @@ def get_studio_updates(live):
     plots = {k: {"data": v} for k, v in plots.items()}
 
     return metrics, params, plots
+
+
+def get_studio_token(repo):
```

Comment:
I wonder if this logic should be in dvc_studio_client

Reply:
What about in dvc? It needs the repo to get the right config value.

Reply:
>  It needs the repo to get the right config value.

I was thinking just accepting optional dvc_repo. If it is not None, assume is a DVC repo and try to read the config. Else try to get the env var

> What about in dvc?

It also works. It is just the same arguments about having dvc-studio-client at all.
If you put it in DVC, we need to pin and require a newer DVC version which is more restrictive for users that if we put it in the client and bumpt it here and in DVC

Reply:
Added https://github.com/iterative/dvc-studio-client/pull/35 to support this in both dvc and dvclive.